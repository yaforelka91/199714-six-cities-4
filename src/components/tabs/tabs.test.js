import React from 'react';
import renderer from 'react-test-renderer';
import Tabs from './tabs.jsx';
import {Router} from 'react-router-dom';
import history from '../../history.js';

const tabList = [`item 1`, `item 2`, `item 3`];

describe(`TabsSnapshot`, () => {
  it(`should render Tabs`, () => {
    const tree = renderer.create(
        <Router history={history}>
          <Tabs
            activeItem='item 1'
            items={tabList}
            onActiveChange={() => {}}
            onCityNameClick={() => {}}
          />
        </Router>
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
