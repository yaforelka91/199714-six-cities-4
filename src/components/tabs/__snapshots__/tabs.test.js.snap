// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabsSnapshot should render Tabs 1`] = `
<div
  className="tabs"
>
  <section
    className="locations container"
  >
    <ul
      className="tabs__list"
    >
      <li
        className="locations__item"
      >
        <a
          className="locations__item-link tabs__item tabs__item--active"
          href="/"
          onClick={[Function]}
        >
          <span>
            item 1
          </span>
        </a>
      </li>
      <li
        className="locations__item"
      >
        <a
          className="locations__item-link tabs__item"
          href="/"
          onClick={[Function]}
        >
          <span>
            item 2
          </span>
        </a>
      </li>
      <li
        className="locations__item"
      >
        <a
          className="locations__item-link tabs__item"
          href="/"
          onClick={[Function]}
        >
          <span>
            item 3
          </span>
        </a>
      </li>
    </ul>
  </section>
</div>
`;
